css/main.css

```diff
@@ -1,3 +1,243 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+/* Header */
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+body {
+	font-family: 'Roboto', sans-serif;
+	-webkit-font-smoothing: antialiased;
+	font-family: 'Lato', sans-serif;
+    background-color: #eee;  
+}  
+#page-wrapper {
+	position: relative;
+}
+li {
+	list-style: none;
+}
+a {
+	color: Black;
+	text-decoration: none;
+}
+
+
+.grid {
+  display: flex;
+}
+header {
```

Comment:
Watch out with your header - something is taking more space than it is given on mobile, and it breaks the layout a little. Can you fix it so it fits on all devices?